content/api/webhooks.apib

```diff
@@ -4,54 +4,81 @@ description: Manage webhooks, which enable you to receive push updates of the ra
-| active            | boolean | Describes status of the webhook | no | Defaults to true, when false the target will no longer receive batches |
-| custom_headers    | JSON   | Object of custom headers to be used during POST requests to target | no | example: `{"x-api-key": "abcd"}` |
-| auth_type         | string | Type of authentication to be used during POST requests to target | no | examples: `none`, `basic`, `oauth2` |
-| auth_request_details | JSON | Object containing details needed to request authorization credentials, as necessary | no | example: `{ "url": "https://oauth.myurl.com/tokens", "body": { "client_id": "<oauth client id>", "client_secret": "<oauth client secret>" }}`|
-| auth_credentials         | JSON | Object containing credentials needed to make authorized POST requests to target | no | examples: `{ "access_token": "<oauth token>", expires_in: 3600 }`, `{ "username": "basicauthuser", "password": "mypassword" }` |
-| auth_token        | string | Authentication token to present in the `X-MessageSystems-Webhook-Token` header of POST requests to target | no | Use this token in your target application to confirm that data is coming from the Webhooks API. <br />example: `5ebe2294ecd0e0f08eab7690d2a6ee69`<br /><br /><span class="label label-danger"><strong>Deprecated</strong></span> in favor of the `auth_type` field. |
 
-__**The SparkPost webhooks API uses MaxMind software [MaxMind License](https://www.maxmind.com/download/geoip/database/LICENSE.txt)**__
+### Authentication
+
+You can use authentication to protect your webhook consumer and verify that only SparkPost is sending you data. We support both OAuth 2 and Basic authentication. When using OAuth 2, you need to set `auth_request_details`. The `auth_credentials` will be set by automatically from this. If you use Basic authentication, you need to set `auth_credentials` with a valid username and password.
+
+## Webhooks Object
+
++ Data Structure: Attributes
+    + name (string) - User-friendly name for webhook
+    + target (string) - URL of the target to which to POST event batches. 
+    + events (array)  - Array of event types this webhook will receive. Use the [Events Documentation](#webhooks-get-events-documentation-get) endpoint to list the available event types.
+    + active (boolean) - The status of the webhook. When false, the target will no longer receive batches.
+    + custom_headers (object) - Object of custom headers to be used during POST requests to target.
+    + auth_type (enum) - Type of authentication to be used during POST requests to target.
+        + none
+        + basic
+        + oauth2
+    + auth_request_details (object) - Object containing details needed to request authorization credentials for OAuth 2.
+    + auth_credentials (object) - Object containing credentials needed to make authorized POST requests for Basic Authentication.
```

Comment:
This is also used for Oauth2 according to the Authentication section. It should mention that and explain what each time would look like (or lean on the Authentication section for explanation)